### PR TITLE
change status

### PR DESCRIPTION
**Pull request by federicazanca**

### CODE REVIEW

**Comment by ElliottKasoar** [comment]
Should this also be a new aiida-mlip major version, if we're saying it's no longer in alpha?

**Comment by alinelena** [comment]
> Should this also be a new aiida-mlip major version, if we're saying it's no longer in alpha?

once we are happy and make a little bit of real life testing... but before 20.

**Review by alinelena — APPROVED** [review]

File: docs/source/user_guide/tutorial.rst [hunk 1]
```diff
@@ -77,7 +77,7 @@ The calculation must be set:
     from aiida.plugins import CalculationFactory
     geomoptCalculation = CalculationFactory("janus.opt")
```
Comment (alinelena) [inline docs/source/user_guide/tutorial.rst:78]:
```suggestion
    geomoptCalculation = CalculationFactory("mlip.opt")
```

Comment (alinelena) [reply docs/source/user_guide/tutorial.rst:78]: be sure you grep for janus and change the entry points

File: pyproject.toml [hunk 2]
```diff
@@ -30,10 +30,10 @@ python = "^3.9"
 aiida-core = "^2.5"
 ase = "^3.22.1"
 voluptuous = "^0.14"
+janus-core = "^v0.4.0b0"
```
Comment (alinelena) [inline pyproject.toml:33]:
```suggestion
janus-core = "^v0.5.0b0"
```

**Review by alinelena — DISMISSED** [review]

**Review by alinelena — APPROVED** [review]

**Pull request merged**